src/uu/ls/src/ls.rs

```diff
@@ -1132,10 +1121,14 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 }
 
 pub fn uu_app() -> Command {
+    // Ensure localization is initialized
+    let _ = uucore::locale::setup_localization("ls");
```

Comment:
Is this necessary? So far it's the first util I have seen that uses `setup_localization`.